src/components/whiteButton/whiteButton.js

```diff
@@ -0,0 +1,22 @@
+const HallOfFame = document.createElement('button')
```

Comment:
I would like that you to use custom element convention as in Portrait component.

Change name od element to HallOfFameButton as it is only a button and HallOfFame will be a separate view.

Reply:
also the name of file should be equal to that